assets/prototype/domain/shared/predicates/datetimes/filters/aboveCapacity/index.test.ts

```diff
@@ -0,0 +1,78 @@
+import aboveCapacity from './index';
```

Comment:
I've splitted up things because the other test started growing too much.
And on the other hand I've needed here to test just the `aboveCapacity`, not necessarily `useDatesListFilterState`